src/container-queries/style.css

```diff
@@ -87,6 +87,7 @@
       border-color: var(--border-color, var(--blue));
       grid-area: button;
       place-self: end;
+      transition: background-color 250ms ease-out;
```

Comment:
This has a brief splash of black when the color changes, so it doesn't appear so smooth. I still need to play around with this some more, I think to account for some of the custom properties in the gradient. I think we might need to use @property to get these to animate @stacyk?

Reply:
The issue is trying to animate from a gradient (image) to a color. Flash to black is the gradient being removed and revealing the color behind it.

Using `@property`, we could animate the two color-stop colors rather than animating the background directly.

Alternate approach would be some way of animating-out the gradient image position while also animating the background color.